test/streamingReceiver.spec.ts

```diff
@@ -247,4 +249,166 @@ describe("ReceiveBatch from Queue/Subscription", function(): void {
 
     await receiveListener.stop();
   });
+
+  it("Abandoned message is retained in the Queue with incremented deliveryCount. After 10 times, you can only get it from the dead letter queue.", async function(): Promise<
+    void
+  > {
+    await queueClient.sendBatch(testMessages);
+
+    let peekedMsgs = await queueClient.peek(2);
+    should.equal(peekedMsgs.length, 2);
+    should.equal(peekedMsgs[0].deliveryCount, 0);
+    should.equal(peekedMsgs[1].deliveryCount, 0);
```

Comment:
Are the above checks needed? When you start receiving the messages you check for the deliverycount, so we can probably skip the peeking above